docs/how-to-guides/isis-developer-guides/refactor-old-isis-apps-tests-tutorial.md

```diff
@@ -0,0 +1,142 @@
+# Tutorial to Refactor Old ISIS Apps and Tests
+
+## 1. Create a Fork of ISIS
+First, on the [main ISIS3 GitHub repo page](https://github.com/DOI-USGS/ISIS3), create a fork of ISIS by clicking on **Fork** at the top of the page. This will take you to a page titled **Create a new fork** where you can specify the **Owner** (your account name) and the repository name (which you can leave as *ISIS3*), then hit the green **Create fork** button at the bottom. This will take you to your new forked ISIS3 repo under your account.
+
+Next, hit the green **<> Code** dropdown option and copy the **SSH** URL under the **Local** tab. 
+
+Then, open your terminal and go to the directory where your other repos reside, type:
+
+```
+git clone --recurse-submodules <paste-repo-link>
+```
+
+Navigate into the newly cloned directory with `cd ISIS3`.
+
+???+ note 
+
+    If you forget to add the `--recurse-submodules` option, you can get the submodules after cloning with this:
+
+    ```
+    git submodule update --init --recursive
+    ```
+
+## 2. Build ISIS in your dev environment
+
+### Create your conda environment
+
+??? note "For MacOS Apple Silicon (M1, M2, etc.) platforms"
+    Before you create your conda environment, you must specify the architecture since some of our software and other scientific dependencies have not yet released builds compatible with the Apple Silicon platforms:
+    ```
+    export CONDA_SUBDIR=osx-64
+    ```
+
+    You can also set this configuration permanently after creating and activating your conda environment, then:
+    ```
+    conda config --env --set subdir osx-64
+    ```
+
+
+In the root of your `ISIS3` repo, create a dedicated conda environment:
+
+```
+conda env create -n <environment-name> -f environment.yml
+```
+
+Make sure you activate your environment after creation:
+
+```
+source activate <environment-name>
+```
+
+
+### Build ISIS
+In the root of your `ISIS3` repo, create a `build` directory:
+
+```
+mkdir build
+```
+
+Then, set the following environment variables:
+
+```
+export ISISROOT=</path/to/your/ISIS3/build>
+export ISISDATA=</path/to/your/data/directory>
+export ISISTESTDATA=</path/to/your/test/data/directory>
+```
+
+Next, build ISIS inside of your `build` directory:
+
+```
+cd build
+cmake -DJP2KFLAG=OFF -GNinja </path/to/your/ISIS3/isis>
+ninja -j24
+```
```

Comment:
It seems like everything above this point is a copy of the first three sections from here:
https://github.com/DOI-USGS/ISIS3/wiki/Developing-ISIS3-with-cmake

Should we just point people to those first three sections?